src/services/__tests__/email_service.test.ts

```diff
@@ -0,0 +1,27 @@
+import { emailService } from '../email_service';
+import { VisitorEmailTemplate } from '../templates';
+const emailOptions = {
+  to: 'hi@hello.com',
+  from: 'bye@seeya.com',
+  templateId: VisitorEmailTemplate.WELCOME_CB,
+  templateModel: {},
+};
+
+describe('Email Service', () => {
+  const email = emailService.init({ apiKey: 'POSTMARK_API_TEST' });
```

Comment:
Do not hardcode this. What if the test API key changes, or we change the email provider? You'll have to change this value in every test. We run the tests with `NODE_ENV=testing`. Just get the config object like we used to in the visitor app tests, and pass in the relevant fragments.

Reply:
Lol, fair. I was just being lazy.